models/chainlink/bnb/chainlink_bnb_ocr_fulfilled_transactions.sql

```diff
@@ -0,0 +1,53 @@
+{{
+  config(
+    tags=['dunesql'],
+    alias='ocr_fulfilled_transactions',
```

Comment:
apply alias macro on all alias properties:
`alias = alias('<tablename>'),`
similar to approach on legacy files, but we default the second param to false to indicate dunesql